contracts/Core/AssetManager.sol

```diff
@@ -6,6 +6,7 @@ import "../IDelegator.sol";
 import "./storage/AssetStorage.sol";
 import "./parameters/child/AssetManagerParams.sol";
 import "./StateManager.sol";
+import "hardhat/console.sol";
```

Comment:
pls remove this if not needed

Reply:
Removed https://github.com/razor-network/contracts/pull/628/commits/8b57244bb0715620c13670fa9bb144b50ef6fe8a